src/config/settings.ts

```diff
@@ -429,7 +429,8 @@ const settings: Settings = {
     pools: {
       [ChainsIds.ETH]: {
         [DexProtocolName.UNISWAP_V3]: {
-          helperContractId: <string>process.env['UNISWAPV3_HELPER_CONTRACT_ID'],
+          helperContractId: <string>process.env.UNISWAPV3_HELPER_CONTRACT_ID,
```

Comment:
```suggestion
          helperContractAddress: <string>process.env.ETH_UNISWAPV3_HELPER_CONTRACT_ADDRESS,
```

contract ID is old, not valid name. let's use address

Reply:
Done